lms/djangoapps/instructor_task/tasks.py

```diff
@@ -81,6 +82,19 @@ def rescore_problem(entry_id, xmodule_instance_args):
 
 
 @task(base=BaseInstructorTask)  # pylint: disable=not-callable
+def override_problem_score(entry_id, xmodule_instance_args):
+    """
+    Overrides a specific learner's score on a problem.
+    """
+    # Translators: This is a past-tense verb that is inserted into task progress messages as {action}.
+    action_name = ugettext_noop('overridden')
+    update_fcn = partial(override_problem_score_module_state, xmodule_instance_args)
```

Comment:
I have never seen this abbreviation for "function" before (I see it in other places in this file).  "fn" or "func" tend to be more common.